### PR TITLE
Feature/us13534 album page

### PR DESCRIPTION
**Pull request by galiotoweb**

### CODE REVIEW

File: _layouts/album.html [hunk 1]
```diff
@@ -2,4 +2,74 @@
...
-{{ content }}
+<div class="media-music-album">
+  <div class="jumbotron jumbotron-xl" style="background-image: url('{{ page.bg_image | imgix: site.imgix }}?{{ site.imgix_params.placeholder }}')" data-optimize-bg-img>
+    <div class="bg-overlay"></div>
+    <div class="jumbotron-content">
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-8 col-sm-offset-2 text-center text-xs-left">
+            <!-- album image -->
+            <img src="{{ page.image }}">
```
Comment (lukemoderwell) [inline _layouts/album.html]: You might consider dropping this image into it's own restrained container. Looks like the album is narrower than the width of the text. Looks like `col-sm-6 col-sm-offset-3` within another row might do the trick.

File: _layouts/album.html [hunk 2]
```diff
@@ -2,4 +2,74 @@
...
+              <li><a href="/music">Music</a></li>
+            </ol>
+
+            <!-- album title -->
+            <h1 class="section-header text-uppercase text-white flush-top">
+              {{ page.title }}
+            </h1>
+
+            <!-- album description -->
+            {{ content }}
```
Comment (lukemoderwell) [inline _layouts/album.html]: text size/color here seems off.

File: _includes/_song-card.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
   </a>
```
Comment (lukemoderwell) [inline _includes/_song-card.html]: you might use the 16x9 placeholder on these song images

File: _includes/_external-music-links.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{% if page.spotify_url.size > 1 or page.apple_music_url.size > 1 or page.youtube_url.size > 1 or page.google_play_url.size > 1 %}
+  <div>
+    <h3 class="text-lowercase font-size-large font-family-base-light text-white ">Stream or Buy</h3>
+    <ul class="list-inline icons narrow">
+      {% if page.spotify_url.size > 1 %}
+        <li><a href="{{ page.spotify_url }}"><img src="//dz6ito7tas43n.cloudfront.net/images/spotify.svg" alt="spotify"></a></li>
+      {% endif %}
+      {% if page.apple_music_url.size > 1 %}
+        <li><a href="{{ page.apple_music_url }}"><img src="//dz6ito7tas43n.cloudfront.net/images/if_itunes_A_104832.svg" alt="itunes"></a></li>
```
Comment (lukemoderwell) [inline _includes/_external-music-links.html]: Not really a blocking comment but each of these svgs is a request to a third party vs being included in the project. Even though they are small requests, I would vote to keep them in the project whether that means including them in our 1 svg file or as their own svgs.

File: _assets/stylesheets/_podcast.scss [hunk 5]
```diff
@@ -32,28 +32,6 @@
...
-    clear: both;
-    display: block;
-  }
-
-  img {
-    height: 33px;
-  }
-}
-
  // 🛠 Replace with variables when crds-styles is added
```
[the remark this answers is not in the document]
Comment (galiotoweb) [reply _assets/stylesheets/_podcast.scss]: Not sure what you mean. I completely refactoring the podcast page to use an inline unordered list and moved any sizing properties to `global.scss` as `.icons`

[the remark this answers is not in the document]
Comment (lukemoderwell) [reply _assets/stylesheets/_podcast.scss]: Was referring to the vars section below. Could you just use `section-header`?

File: _layouts/album.html (hunk 2 above)
Comment (galiotoweb) [reply _layouts/album.html]: Good catch. I added a class for this, and fixed it on the song-detail page as well.

File: _includes/_song-card.html (hunk 3 above)
Comment (galiotoweb) [reply _includes/_song-card.html]: Great. Thanks!

File: _layouts/album.html (hunk 1 above)
Comment (galiotoweb) [reply _layouts/album.html]: Thanks, forgot to add imgix here as well.

File: _assets/stylesheets/_podcast.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (galiotoweb) [reply _assets/stylesheets/_podcast.scss]: These styles are only for the podcast page, and we can change this when we go through and clean up CSS throughout. It doesn't affect this story except for the bit I moved to `global.scss`

File: _includes/_external-music-links.html (hunk 4 above)
Comment (lukemoderwell) [reply _includes/_external-music-links.html]: @galiotoweb This is a really good explanation. Learning new things!

**Pull request merged**